server/services/profanityCheck.js

```diff
@@ -0,0 +1,15 @@
+const fs = require('fs');
+const wordListPath = require('word-list');
+const profanities = fs.readFileSync(wordListPath, 'utf8').split('\n');
+
+var notProfanity = function(text){
+    var returnVal = false; 
+    for (var i = 0; i < profanities.length; i++) {
+        if(text.toLowerCase().includes(profanities[i].toLowerCase())){
+            returnVal = true;
+            break;
+        }
+    }
+    return returnVal;
```

Comment:
This looks fine, even if I don't love `for` loops in JS! A small change you could do though, is to get rid of `returnVal`. Just return `true` in the `if` statement and return `false` at the end of the function.

Reply:
have turned it into a one liner